packages/serverless/settings/common/index.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+const GENERAL_SETTINGS = [
+  'csv:quoteValues',
+  'dateFormat:dow',
+  'dateFormat:scaled',
+  'dateFormat:tz',
```

Comment:
It's probably a NIT, but I'm somewhat concerned about having this list of setting IDs being fully decoupled from the actual setting definitions. 

I'm a bit scared of human error coming from developers not knowing about this list when they, say, add a new setting, or remove an existing one.

Do we have anything in mind to avoid the risk of having this list "desync" from the actual definitions?

Reply:
I'd like this to be in a package, rather than coded as strings.

Reply:
We could have the ids of the settings in a package, and then imported/used both by the allowlists and by the actual settings definitions.

Or maybe the setting definitions themselves could be moved into packages and then the allowlist importing the ids from here?

Either way it's a lot of "moving code", and not a blocker for the initial implementation

Reply:
Both approaches make sense to me. I think the second one would require significantly more "moving code" changes so we might need to defer it to a separate PR, but the first approach could be implemented as part of this PR - or at least for now we can make packages for the ids and use them in the allowlists, and in a later PR to import them in all places with definitions where the settings are registered.